docs/template_plugin/tests/functional/op_reference/tile.cpp

```diff
@@ -64,38 +64,112 @@ TEST_P(ReferenceTileTest, CompareWithRefs) {
+            Tensor(ET_INT, {2}, std::vector<T_INT>{2, 1}),
+            Tensor(ET, {2, 2, 3}, std::vector<T>{
+                1.1f, 2.2f, 3.3f, 1.1f, 2.2f, 3.3f, 4.4f, 5.5f, 6.6f, 4.4f, 5.5f, 6.6f
+            }),
+            "tile_3d_to_3d_repeats_broadcast_float_val"),
     };
     return params;
 }
 
 std::vector<TileParams> generateCombinedParams() {
     const std::vector<std::vector<TileParams>> generatedParams {
-        generateParams<element::Type_t::i8>(),
-        generateParams<element::Type_t::i16>(),
-        generateParams<element::Type_t::i32>(),
-        generateParams<element::Type_t::i64>(),
-        generateParams<element::Type_t::u8>(),
-        generateParams<element::Type_t::u16>(),
-        generateParams<element::Type_t::u32>(),
-        generateParams<element::Type_t::u64>(),
-        generateParams<element::Type_t::bf16>(),
-        generateParams<element::Type_t::f16>(),
-        generateParams<element::Type_t::f32>(),
-        generateParams<element::Type_t::f64>(),
+        // test each data type for fixed repeats type
+        generateParams<element::Type_t::i8, element::Type_t::i32>(),
+        generateParams<element::Type_t::i16, element::Type_t::i32>(),
+        generateParams<element::Type_t::i32, element::Type_t::i32>(),
+        generateParams<element::Type_t::i64, element::Type_t::i32>(),
+        generateParams<element::Type_t::u8, element::Type_t::i32>(),
+        generateParams<element::Type_t::u16, element::Type_t::i32>(),
+        generateParams<element::Type_t::u32, element::Type_t::i32>(),
+        generateParams<element::Type_t::u64, element::Type_t::i32>(),
+        generateParams<element::Type_t::bf16, element::Type_t::i32>(),
+        generateParams<element::Type_t::f16, element::Type_t::i32>(),
+        generateParams<element::Type_t::f32, element::Type_t::i32>(),
+        generateParams<element::Type_t::f64, element::Type_t::i32>(),
+        generateParams<element::Type_t::i8, element::Type_t::i64>(),
+        generateParams<element::Type_t::i16, element::Type_t::i64>(),
+        generateParams<element::Type_t::i32, element::Type_t::i64>(),
+        generateParams<element::Type_t::i64, element::Type_t::i64>(),
+        generateParams<element::Type_t::u8, element::Type_t::i64>(),
+        generateParams<element::Type_t::u16, element::Type_t::i64>(),
+        generateParams<element::Type_t::u32, element::Type_t::i64>(),
+        generateParams<element::Type_t::u64, element::Type_t::i64>(),
+        generateParams<element::Type_t::bf16, element::Type_t::i64>(),
+        generateParams<element::Type_t::f16, element::Type_t::i64>(),
+        generateParams<element::Type_t::f32, element::Type_t::i64>(),
+        generateParams<element::Type_t::f64, element::Type_t::i64>(),
+        // test each repeats type for fixed data type
+        // generateParams<element::Type_t::f32, element::Type_t::u4>(), // fails according to *-71891
+        generateParams<element::Type_t::f32, element::Type_t::u8>(),
```

Comment:
Why do we have u8/i8/u32/... only for f32?

Reply:
I guess it's too much to test Cartesian product of all available cases, so I decided to limit it by covering each `input` case for default `repeats` and each `repeats` case for default `input`

Reply:
Or do you mean also cover small number of other cases (like f64, i32, i64)?

Reply:
Added all types except boolean and u1